src/protocols/dns.cpp

```diff
@@ -194,8 +194,6 @@ namespace dns {
  * This is the internal object thanks to which Request is movable. Of
  * course, RequestImpl is not movable, since its address is passed to
  * one of the many evnds delayed requests functions.
- *
- * \remark You should not use this class directly, use Request instead.
```

Comment:
Why was this removed? Is it the case that now people should use this class directly?

I think that should not be the case.